autogen/oai/client.py

```diff
@@ -142,7 +151,10 @@ def _client(self, config, openai_config):
         """
         openai_config = {**openai_config, **{k: v for k, v in config.items() if k in self.openai_kwargs}}
```

Comment:
Is there a reason you're leaving this named 'openai_config' if it's also being used for the GeminiClient? Could be confusing in the future :)